Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2014,8 +2027,8 @@ public void getShortUrl(int type, Collection<String> tags, String channel, Strin
      * @see BranchLinkData#putParams(String)
      * @see BranchLinkData#putDuration(int)
      * @see BranchLinkCreateListener
-     *
-     * @deprecated use {@link BranchShortLinkBuilder} instead.
+     * @deprecated Use {@link BranchUniversalObject } to create links to your content.
```

Comment:
I'd prefer to wait a version or two before we have all of the documentation completed. Please do not deprecate anything yet.